client/src/pages/ProfileDetails/BookingForm/BookingForm.tsx

```diff
@@ -41,15 +41,25 @@ const BookingForm = (): JSX.Element => {
 
   useEffect(() => {
     if (id) {
-      fetchProfile(id).then((data) => setAvailableTime(data.success.profile.availableTime));
+      fetchProfile(id).then((data) => {
+        if (data.success) {
+          setAvailableTime(data.success.profile.availableTime);
+        } else {
+          console.log(data.error);
```

Comment:
You can update the Snackbar with the error message rather than logging it to the console.